content/references/protocol-reference/ledger-data/ledger-object-types/accountroot.md

```diff
@@ -34,21 +34,21 @@ The `AccountRoot` object type describes a single [account](accounts.html), its s
 
 ## {{currentpage.name}} Fields
 
-The `AccountRoot` object has the following fields:
+An `AccountRoot` object has the following fields:
 
 | Field                         | JSON Type | [Internal Type][] | Required? | Description  |
 |:------------------------------|:----------|:------------------|:----------|:-------------|
 | `Account`                     | String    | AccountID         | Yes       | The identifying (classic) address of this [account](accounts.html). |
 | `AccountTxnID`                | String    | Hash256           | No        | The identifying hash of the transaction most recently sent by this account. This field must be enabled to use the [`AccountTxnID` transaction field](transaction-common-fields.html#accounttxnid). To enable it, send an [AccountSet transaction with the `asfAccountTxnID` flag enabled](accountset.html#accountset-flags). |
 | `Balance`                     | String    | Amount            | No        | The account's current [XRP balance in drops][XRP, in drops], represented as a string. |
-| `BurnedNFTokens`              | Number    | UInt32            | No        | How many total of this account's issued [non-fungible tokens](non-fungible-tokens.html) have been burned. This number is always equal or less than `MintedNFTokens`. |
+| `BurnedNFTokens`              | Number    | UInt32            | No        | How many total of this account's issued [non-fungible tokens](non-fungible-tokens.html) have been burned. This number is always equal or less than `MintedNFTokens`. _(Added by the [NonFungibleTokensV1_1 amendment][])_ |
```

Comment:
We should add some notes to our style guide about how to mention amendments. My thoughts are:

- Use "Added by" if the amendment is enabled on Mainnet and its pre-amendment functionality has not been retired.
- If the amendment is retired (or obsolete) in the latest stable release, remove the note entirely.
- Use "Requires" or "If the (name) amendment is enabled" along with the `:not_enabled:` icon if the amendment is released but not currently enabled
- TBD for pre-amendment features in development like AMM; most likely these are going to stop being on xrpl.org soon. But for the ones that are already there now, use a note plus the `:not_enabled:` icon.

Reply:
👍  in-development unreleased features won't be documented on references like accountroot. However, we should link to the relevant [XRPL-Standards](https://github.com/XRPLF/XRPL-Standards) discussion(s) and specs on the places where XRPL.org visitors may be interested in learning more about those potential upcoming features. Basically, the actual content will live separately, but it will still be easy to find because we'll be liberal with providing links to that content.